TombEngine/Objects/TR1/Trap/SlammingDoors.cpp

```diff
@@ -0,0 +1,68 @@
+#include "framework.h"
+#include "Objects/TR1/Trap/SlammingDoors.h"
+
+#include "Game/control/Box.h"
+#include "Game/effects/effects.h"
+#include "Game/Lara/lara.h"
+#include "Math/Objects/Vector3i.h"
+#include "Math/Random.h"
+
+using namespace TEN::Math;
+
+namespace TEN::Entities::Traps::TR1
+{
+	constexpr auto SLAMMING_DOORS_DAMAGE = 400;
+	constexpr auto LARA_RADIUS = 100;
```

Comment:
This constant already exists in `lara.h`.

Reply:
And LARA_RADIUS is unused in this code now, because the blood spawning is sorted by the trap collision function.